src/ape_init/_cli.py

```diff
@@ -7,6 +7,32 @@
 from ape.utils import github_client
 
 
+def read_dependencies_from_toml(file_path):
+    with open(file_path, "r") as file:
+        content = file.readlines()
+
+    in_dependencies = False
+    dependencies = []
+
+    for line in content:
+        stripped_line = line.strip()
```

Comment:
This should be using the standard tomllib, I would think:
https://docs.python.org/3/library/tomllib.html

Reply:
Only in 3.11+, for older versions this is the recommended backport: https://github.com/hukkin/tomli